### DISCUSSION TITLE
Bounding box numbers

### DISCUSSION THREAD

**Discussion by levidurfee** [blakeblackshear/frigate#8743] [Ask A Question] [answered]
On the camera debug page with "Bounding box" enabled, it shows the object type, a percentage, then another number like the example below. 

What does the number after the percentage indicate?

## example

```
person: 54% 96
car: 60% 494
car: 59% 368
```

Thanks!

**Comment by NickM-27** [comment, marked as answer]
that number represents the area of the box in pixels

**Reply by levidurfee** [reply]
Thank you!

**Comment by Kersh72** [comment]
Can we remove the last number being displayed? I really don't care how many pixels my person is.

**Reply by NickM-27** [reply]
having more information does not hurt anything, the bounding boxes are there for debugging purposes
